src/Client/ClientBase.cpp

```diff
@@ -2956,6 +2956,7 @@ void ClientBase::init(int argc, char ** argv)
     /// Common options for clickhouse-client and clickhouse-local.
     options_description.main_description->add_options()
         ("help", "produce help message")
+        ("verbose", "print query and other debugging info")
```

Comment:
L. 2958: `("help", "print usage summary, combine with --verbose to display all options")`